__tests__/app.test.js

```diff
@@ -49,10 +49,43 @@ describe("GET /api", () => {
         return Promise.all([body, endpoints]);
       })
       .then(([body, endpoints]) => {
-        console.log(body);
-        console.log(endpoints);
         const parsedEndpoints = JSON.parse(endpoints);
         expect(body).toEqual({ endpoints: parsedEndpoints });
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET 200 sends the article corresponsing to the ID given", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(article.article_id).toBe(1);
+        expect(typeof article.body).toBe("string");
+        expect(typeof article.topic).toBe("string");
+        expect(typeof article.created_at).toBe("string");
+        expect(typeof article.votes).toBe("number");
+        expect(typeof article.article_img_url).toBe("string");
+      });
```

Comment:
As we are testing a single object, you can also test the values of the other properties as well.